src/Common/src/System/Xml/XmlConvertEx.cs

```diff
@@ -66,7 +66,7 @@ public static string[] SplitString(string value)
 
         public static Uri ToUri(string s)
         {
-            if (s != null && s.Length > 0)
```

Comment:
it would have to be 'if ( s?.Length ?? 0 > 0) ;' because the return value of 's?.Length' is a nullable